packages/vaults/src/transaction/policy/policy_create.ts

```diff
@@ -8,13 +8,38 @@ import {TransactionMapperAbstract} from "../transaction_mapper";
 import {TransactionRequest} from "../transaction_request";
 import {RequestMapperAbstract} from "../request_mapper";
 
-
+/**
+ * Interface for a transaction that creates a new transfer policy.
+ * The policy determines the number of necessary signatures for executing
+ * the transfer transaction depending on the threshold amount.
+ * This transaction can only be requested or approved only by users with the admin role.
+ * This transaction can be executed in a batch.
```

Comment:
This transaction can only be requested or approved by users with the admin role.
This transaction can be executed in a batch, which means that all transactions marked with a single batch_id are executed or rejected together.